File: NYTimes/app/src/main/java/com/example/nytimes/views/MakeAccountActivity.java
package com.example.nytimes.views;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.example.nytimes.R;

public class MakeAccountActivity extends AppCompatActivity {
    private TextView Notnow;
    private  Button account;
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_make_account);
        Notnow=findViewById(R.id.btnNotNow);
        account=findViewById(R.id.btnFreeAccount);
        Notnow.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent( MakeAccountActivity.this,MainActivity.class);
                startActivity(intent);
                finish();
            }
        });

        account.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent( MakeAccountActivity.this,SignUpActivity.class);
                startActivity(intent);
                finish();
            }
        });
    }
}

File: NYTimes/app/src/main/java/com/example/nytimes/views/MakeAccountActivity.kt
package com.example.nytimes.views

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.nytimes.R
import kotlinx.android.synthetic.main.activity_make_account.*

class MakeAccountActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_make_account)

        btnNotNow.setOnClickListener(View.OnClickListener {
            val intent = Intent(this@MakeAccountActivity, MainActivity::class.java)
            startActivity(intent)
            finish()
        })

        btnFreeAccount.setOnClickListener(View.OnClickListener {
            val intent = Intent(this@MakeAccountActivity, SignUpActivity::class.java)
            startActivity(intent)
            finish()
        })
    }
}